src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginDiscoverer.cs

```diff
@@ -63,6 +63,17 @@ internal class TestPluginDiscoverer
 
 #if !WINDOWS_UAP
 
+            if(!extensionPaths.Any())
+            {
+                // C++ UWP adaters do not follow TestAdapater naming convention, so making this exception)
+                extensionPaths = extensionPaths.Concat<string>(new List<string> { "Microsoft.VisualStudio.TestTools.CppUnitTestFramework.ComInterfaces.dll",
```

Comment:
Where are we shipping these dlls? Extensions folder?

Reply:
Where are we shipping these dlls? Extensions folder?
If in Extensions folder, then we need not to do this explicitly.